glide.lock

```diff
@@ -50,7 +43,7 @@ imports:
   subpackages:
   - proto
 - name: github.com/google/cadvisor
-  version: a8085bf9276c22f16dbcd7aa56f0d4d0626a0b2e
+  version: aa6f80814bc6fdb43a0ed12719658225420ffb7d
```

Comment:
can you change this back to the old version? same with the below `k8s.io/kubernetes` dep